clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/mark_node_failed.py

```diff
@@ -12,14 +12,10 @@ def make_key(site, node_type, datatore):
     else:
         return "/clearwater/{}/{}/clustering/{}".format(site, node_type, datastore)
 
-root_log = logging.getLogger()
-root_log.setLevel(logging.INFO)
-
-ch = logging.StreamHandler(sys.stdout)
-ch.setLevel(logging.DEBUG)
-formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-ch.setFormatter(formatter)
-root_log.addHandler(ch)
+print "Detailed output being sent to mark_node_failed.log"
+logging.basicConfig(filename="mark_node_failed.log",
```

Comment:
I think this would mean it needs to be run with 'sudo' - are you happy with that?